src/components/step-sequence/step-sequence.stories.mdx

```diff
@@ -7,6 +8,12 @@ import * as stories from "./step-sequence.stories.tsx";
 
 # StepSequence
 
+<DeprecationWarning>
+We've deprecated StepSequence. It's also deprecated in the Design System and we recommend that you do not use it in new products.
```

Comment:
suggestion(non-blocking): I wonder if we should just be more to the point here and remove `we`

Very minor and feel free to ignore but maybe something like this:

```suggestion
StepSequence has been deprecated in Carbon and the Design System.
<br />
It is recommended to use the StepFlow component - which is an accessible alternative.
```